### ISSUE TITLE
Accessing data in Climsoft through R

### ISSUE THREAD

**Issue by brasmus** [opencdms/Climsoft#305]
Hi, I'm planning to build an R-shiny-app that reads data  from Climsoft. Is it feasible to access the Climsoft database through R?

**Comment by isedwards** [comment]
Yes, it's certainly possible. Some work has been done on this in the past as part of the first phase of the [SASSCAL](http://www.sasscal.org/) project. They used R/Shiny to connect to a Climsoft 3.1 or 4 database using ODBC.
The code is not currently being developed/maintained, but it would be good to take it forward.  See the code on [GitHub](https://github.com/sasscal-dwd-apps/) and documentation [here](https://sasscal-dwd-apps.github.io/ACD-App/en/documentation.html).

Also, some of the Climsoft development team are involved with [R-Instat](https://github.com/africanmathsinitiative/R-Instat) (a statistics software package powered by R). R-Instat has the ability to connect directly to a Climsoft database in order to provide analysis and products.

Feel free to ask more questions about Climsoft here, or about the SASSCAL or R-Instat projects in their respective GitHub repositories.
